pkgs/servers/http/apache-modules/mod_python/default.nix

```diff
@@ -1,36 +1,32 @@
-{ lib, stdenv, fetchurl, apacheHttpd, python2, libintl }:
+{ lib, stdenv, fetchFromGitHub, apacheHttpd, python3, libintl }:
 
 stdenv.mkDerivation rec {
   pname = "mod_python";
-  version = "3.5.0";
+  version = "unstable-2022-10-18";
 
-  src = fetchurl {
-    url = "http://dist.modpython.org/dist/${pname}-${version}.tgz";
-    sha256 = "146apll3yfqk05s8fkf4acmxzqncl08bgn4rv0c1rd4qxmc91w0f";
+  src = fetchFromGitHub {
+    owner = "grisha";
+    repo = pname;
+    rev = "d066b07564d2194839eceb535485eb1ba0c292d8";
+    hash = "sha256-EH8wrXqUAOFWyPKfysGeiIezgrVc789RYO4AHeSA6t4=";
   };
 
   patches = [ ./install.patch ];
 
-  postPatch = ''
-    substituteInPlace dist/version.sh \
-        --replace 'GIT=`git describe --always`' "" \
-        --replace '-$GIT' ""
-  '';
-
-  installFlags = [ "LIBEXECDIR=${placeholder "out"}/modules" ];
-
-  preInstall = ''
-    mkdir -p $out/modules $out/bin
-  '';
+  installFlags = [
+    "LIBEXECDIR=${placeholder "out"}/modules"
+    "BINDIR=${placeholder "out"}/bin"
```

Comment:
BTW, the included script (`$out/bin/mod_python`) is not working (it depends on `StringIO` that is removed in Python 3), but it is also not working in `master` and since nobody complained yet I don't think it is actually useful.